### PR TITLE
Add iOS accessibility hints to the tabbar "buttons".

### PR DESCRIPTION
**Pull request by jerolimov**
Add iOS accessibility hints to the tabbar "buttons" which is a good thing, obviously.

This makes the tabbar also testable with iOS UITests.

PS: Thanks for sharing the tabbar, nice work!

### CODE REVIEW

**Comment by jerolimov** [comment]
ping @brentvatne

File: DefaultTabBar.js [hunk 1]
```diff
@@ -46,7 +46,13 @@ var DefaultTabBar = React.createClass({
     var activeTextColor = this.props.activeTextColor || "navy";
     var inactiveTextColor = this.props.inactiveTextColor || "black";
     return (
-      <TouchableOpacity style={[styles.tab]} key={name} onPress={() => this.props.goToPage(page)}>
+      <TouchableOpacity
+        key={name}
+        accessible={true}
+        accessibilityLabel={name}
```
Comment (skv-headless) [inline DefaultTabBar.js]: I don't see such property http://facebook.github.io/react-native/releases/next/docs/touchablewithoutfeedback.html#props

**Comment by skv-headless** [comment]
Sorry for delay. Before merge it I need some time to understand how it works.

File: DefaultTabBar.js (hunk 1 above)
Comment (jerolimov) [inline DefaultTabBar.js]: They work on every View element: http://facebook.github.io/react-native/releases/next/docs/view.html#content

**Comment by jerolimov** [comment]
Np. To test it you can use the iOS simulator "accessibility" tools. Start the simulator and open the settings app (in the simulator) and enable them unter "General" > "Accessibility" > "Accessibility Inspector".

You can enable / disable the debugging view them with the X in the corner. Then navigate to your ScrollableTabView and compare the the accessibility informations before and after.

It added this now also to the FB example code. When using images this is particularly important.

Here are some screenshots before and after the change:

Before:

simulator screen shot 11 03 2016 13 45 12

simulator screen shot 11 03 2016 13 45 13

After:

simulator screen shot 11 03 2016 13 53 45

simulator screen shot 11 03 2016 13 53 46

**Comment by skv-headless** [comment]
@jerolimov could you please create separate PR for last commit (I'm not sure merge or not)? And I'll merge first commit. If it is not difficult could you please rebase instead of merge. I'm really sorry that it took so long.

**Comment by jerolimov** [comment]
Hey @skv-headless, rebased / cherrypick just the first commit here.

I'm also unsure with the FB example. Here is the separate [commit](https://github.com/jerolimov/react-native-scrollable-tab-view/commit/ffd669947a9c58e6fb4b8a1b31cc0fcbfdf52325).

Another approach would be to introduce more "options" which could be used to identify which tab is used. Maybe standard `key` instead of `tabLabel`. Btw. using the prop "label" inside as name is also confusing, or?

**Pull request merged**